src/components/AddPlayerToScoreboard/AddPlayerToScoreboard.js

```diff
@@ -71,55 +71,62 @@ class AddPlayerToScoreboard extends React.Component {
     return (
       <div className="component-background">
         <div className="add-player-to-scoreboard">
-          <h1 className="score-layout">
-            <div>&nbsp;</div>
-            {this.props.languageVersion?.scoreboardDescription.toUpperCase()}
-            <button
-              className="button"
-              onClick={() => {
-                this.closeScoreboard();
-              }}
-            >
-              X
-            </button>
-          </h1>
-          <div className="podium">
-            <div className="first-place">
+          <div className="row justify-content-center align-items-center">
+            <div>
+              <h2 className="mt-2">
+                {this.props.languageVersion?.scoreboardDescription.toUpperCase()}
+              </h2>
+            </div>
+            <div>
+              <button
+                className="button pull-right"
+                onClick={() => {
+                  this.closeScoreboard();
+                }}
+              >X</button>
```

Comment:
X button is not aligned to left side of the modal. Also in lower resolution X is displayed below header.